package.json

```diff
@@ -34,7 +35,20 @@
       "react": {
         "version": "detect"
       }
-    }
+    },
+    "overrides": [
+      {
+        "files": [
+          "*.ts",
+          "*.tsx"
+        ],
+        "rules": {
+          "comma-dangle": "off",
+          "react/prop-types": "off",
+          "quote-props": "off"
```

Comment:
Two more eslint rules worth disabling now:

```json
"react/jsx-max-props-per-line": "off",
"max-len": "off"
```

I'll plan to add these as I incorporate feedback. 🙏🏼